Modules/_sha3/sha3module.c

```diff
@@ -40,18 +40,18 @@
 #elif PY_BIG_ENDIAN
   /* opt64 is not yet supported on big endian platforms */
   #define KeccakOpt 32
-#elif SIZEOF_VOID_P == 8 && defined(PY_UINT64_T)
+#elif SIZEOF_VOID_P == 8
   /* opt64 works only on little-endian 64bit platforms with unsigned int64 */
   #define KeccakOpt 64
 #else
   /* opt32 is used for the remaining 32 and 64bit platforms */
   #define KeccakOpt 32
 #endif
 
-#if KeccakOpt == 64 && defined(PY_UINT64_T)
+#if KeccakOpt == 64
   /* 64bit platforms with unsigned int64 */
-  typedef PY_UINT64_T UINT64;
-  typedef unsigned char UINT8;
+  typedef uint64_t UINT64;
```

Comment:
Probably we can remove these `typedef`s and restore the original ones?

https://github.com/python/cpython/blob/132acaba5a7f01373ca624b1a5975b190fe866f5/Modules/_sha3/kcp/KeccakP-1600-opt64.c#L21-L24